pvc-basic/020-development-pvc.md

```diff
@@ -0,0 +1,65 @@
+Lets check how to dynamically provision storage using StorageOS.
+
+During cluster bootstrap StorageOS creates a StorageClass called `fast`. A
+StorageClass is a Kubernetes construct that describes different "classes" of
+Storage and their providers.
+
```

Comment:
Maybe add a `kubectl describe sc fast` command here to show what the default StorageClass entails.

Reply:
I thought about that, but the scenario already had a lot of information and show the dev and prod sc, so showing one more was a bit too much IMO.